package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "brainly-style-guide",
-  "version": "218.0.1",
+  "version": "v218.0.1-beta-a11y-buttons.1",
```

Comment:
just placing comment to not forget reverting this change

Reply:
it is reverted :)